### PR TITLE
[new-doc] how to generate and use format.ps1xml file (iss #865)

### PR DESCRIPTION
**Pull request by vladimir-shcherbakov**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by vladimir-shcherbakov** [comment]
issue #865

File: documentation/development-docs/generate-format.ps1xml-file.md [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+                                                                                                                                                 ountName
+--                                   ----                      -----   --------------                       --------                             -----------------
+c9cbd920-c00c-427c-852b-c329e824c3a8 Azure SDK Powershell Test Enabled c9cbd920-c00c-427c-852b-c329e824c3a8 72f988bf-86f1-41af-91ab-7a64d1d63df5
+6b085460-5f21-477e-ba44-4cd9fbd030ef Azure SDK Infrastructure  Enabled 6b085460-5f21-477e-ba44-4cd9fbd030ef 72f988bf-86f1-41af-91ab-7a64d1d63df5
+
+
+```
+
+The default table output reveals some issues:
+1. **SubscriptionId** property values duplicates the **Id** property values,
```
Comment (markcowl) [inline documentation/development-docs/generate-format.ps1xml-file.md]: I think it's important to call out these
- The selected fields don't feit in a standard window
- The columns are not displayed in oirder of importance to the customer for doign their work

File: documentation/development-docs/generate-format.ps1xml-file.md [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+Powershell allows to configure cmdlets output view with the [format.ps1xml](https://msdn.microsoft.com/en-us/library/gg580992) files.
+
+To provide a better PowerShell Azure cmdlets output experience we worked out a mechanism to quickly generate a [format.ps1xml](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_format.ps1xml?view=powershell-6) file:
+
+
+1. Mark all the cmdlet output type public properties that should go to the table output with the *Ps1XmlAttribute* attribute.
+2. Run the New-FormatPs1Xml cmdlet to generate the format.ps1xml file.
+
+---
+ We presume that for the [output type](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/azure-powershell-design-guidelines.md#output-type) you created a new class that, for example,  wraps a returning .NET SDK type, rather to use PSObject.
```
Comment (markcowl) [inline documentation/development-docs/generate-format.ps1xml-file.md]: ..., rather *than* PSObject.

**Comment by vladimir-shcherbakov** [comment]
@markcowl 
The column order in the output table will be the same as the order of the properties in the output class. Does it make sense?

**Pull request merged**